cpp/oneapi/dal/backend/primitives/reducer_dpc_test.cpp

```diff
@@ -0,0 +1,171 @@
+/*******************************************************************************
```

Comment:
Test naming/location does not follow new conventions, see how it is done in for `blas` or `stat` primitives